backend/benefit/applications/services/applications_csv_report.py

```diff
@@ -135,7 +135,7 @@ def CSV_COLUMNS(self):
                 str,
             ),
             CsvColumn(
-                "Yhdistys jolla yritystoimintaa?",
+                "Yhdistys jolla taloudellista toimintaa??",
```

Comment:
Is the double question mark at the end intentional?

Reply:
fixed